### PR TITLE
1.1.2

### PR DESCRIPTION
**Pull request by 6l20**
## PR Description

- enables existingSecret as env
- disables setCap forbidden on Azure EKS
- Add postgres client.key trick chmod 0600
- Enforce condition on vault-certs Secret

## Documentation

- [ ] I thought about documentation and added the `documentation` label to this PR if updates are required.

### CODE REVIEW

File: templates/deployment.yaml [hunk 1]
```diff
@@ -40,15 +40,15 @@ spec:
         {{- $agentImage := printf "%s:%s" .image.repository .image.tag }}
         - name: init-token-{{ .name }}
           securityContext:
-            capabilities:
-              add:
-              - IPC_LOCK
+            runAsUser: 1001
```
Comment (ggarri) [inline templates/deployment.yaml:43]: what it is the reasoning behind this user?

File: templates/init-migrate-hook.yaml [hunk 2]
```diff
@@ -53,16 +37,46 @@ spec:
...
       imagePullSecrets:
         - name: {{ include "quorumkeymanager.imagePullSecretName" . }}
       {{- end }}
+      {{- if .Values.postgresql.tls.enabled }}
+      initContainers:
+        - name: init-chmod-data
+          image: docker.io/bitnami/bitnami-shell:10-debian-10-r185
+          imagePullPolicy: "Always"
+          resources:
+            requests:
```
Comment (ggarri) [inline templates/init-migrate-hook.yaml:53]: why you need to limit the resources?

File: templates/init-migrate-hook.yaml [hunk 3]
```diff
@@ -53,16 +37,46 @@ spec:
...
       restartPolicy: Never
       {{- if .Values.imageCredentials }}
       imagePullSecrets:
         - name: {{ include "quorumkeymanager.imagePullSecretName" . }}
       {{- end }}
+      {{- if .Values.postgresql.tls.enabled }}
+      initContainers:
+        - name: init-chmod-data
+          image: docker.io/bitnami/bitnami-shell:10-debian-10-r185
+          imagePullPolicy: "Always"
```
Comment (ggarri) [inline templates/init-migrate-hook.yaml:51]: I dont think we need to always pull for new images for this

**Review by ggarri — APPROVED** [review]

File: templates/deployment.yaml (hunk 1 above)
Comment (6l20) [reply templates/deployment.yaml:43]: Indeed this is the postgres user probably been set here during a search/replace operation. Not relevant here. I will remove it

File: templates/init-migrate-hook.yaml (hunk 2 above)
Comment (6l20) [reply templates/init-migrate-hook.yaml:53]: Well this was taken from an exemple, as is, that ensures the Pod to be properly scheduled by k8s

File: templates/init-migrate-hook.yaml (hunk 3 above)
Comment (6l20) [reply templates/init-migrate-hook.yaml:51]: Yes right, I will turn it to `IfNotPresent`

**Pull request merged**